packages/SwingSet/test/devices/test-devices.js

```diff
@@ -118,6 +127,10 @@ test.serial('d1', async t => {
   const hostStorage = provideHostStorage();
   await initializeSwingset(config, [], hostStorage, t.context.data);
   const c = await makeSwingsetController(hostStorage, deviceEndowments);
+  for (let i = 0; i < 5; i += 1) {
+    // eslint-disable-next-line no-await-in-loop
+    await c.step(); // vat start
+  }
```

Comment:
this might also fall into the "`c.run()` first" category: do a `c.run` at startup, let all the startup-ish things finish, then do a single `c.step` after the `c.queueToVatRoot`